Demo/UIKit/ComponentsViewController.swift

```diff
@@ -69,7 +75,7 @@ internal final class ComponentsViewController: UIViewController, PresenterExampl
 
     internal func presentCardComponent() {
         if componentsView.isUsingSession {
-            // todo: add card component with session
+            cardComponentExample.presentCardComponentSession()
```

Comment:
Do we really need different names for `presentCardComponentSession` and `presentCardComponent` ?
`cardComponentExample` means "session"..

Or `cardComponentExample` means it will present "CardComponent"  😅

How about something like "start()" ?

Reply:
Good point. I think we can do cardComponentExample.present(). We should also make this change for other classes as well.

Reply:
db9e111e04e4f56af38dde3b108aa2bd16128ba6